vector/src/main/res/layout/activity_vector_invite_members.xml

```diff
@@ -35,4 +26,14 @@
             android:layout_centerVertical="true"
             android:indeterminate="true" />
     </RelativeLayout>
+
+    <com.getbase.floatingactionbutton.FloatingActionButton
+        android:id="@+id/button_start_chat"
```

Comment:
What do you suggest? This was the best I could come up with haha, maybe button_match_user_id_start_chat works better?

Reply:
Oh, ok, I understand now. In fact the FAB is used to create the Room, right? It's not a common way to use FAB

Reply:
Yeah, the main goal is to avoid more than one view for adding users. Right now the app has both the activity and the dialog which can be confusing at first.